src/utils.rs

```diff
@@ -31,6 +32,145 @@ fn _bytes_to_hex(bytes: &[u8]) -> String {
     hex
 }
 
+// Pretty ugly helper to count the number of arguments.
+#[macro_export]
```

Comment:
These `macro_export`s here are pretty ugly and not what we should do. (I know it's my code 😬 but this was the quick and dirty way.). We only want to be able to use this macro in other parts of the library not export it to the outside world. There's already a `macro_use` on `utils` now and the order is correct (I did that for another macro). So you should be able to just remove the export and the corresponding `use`s.